internal/noderesourcetopology/equality.go

```diff
@@ -38,6 +38,22 @@ func EqualZones(zonesA, zonesB nrtv1alpha2.ZoneList, isRebootTest bool) (bool, e
 	zA := SortedZoneList(zonesA)
 	zB := SortedZoneList(zonesB)
 
+	if isRebootTest {
```

Comment:
if we're aiming for checking at the node level, I think we can remove the `isRebootTest` flag, and use this checkas a sanity for all the tests

Reply:
good points, this is valid for any test